apps/checker/test/scala/services/SentenceHelperTest.scala

```diff
@@ -17,4 +17,28 @@ class SentenceHelperTest extends AsyncFlatSpec with Matchers {
       WordInSentence("Cafes, bars, and restaurants will be able to seat 100 indoors and 200 outdoors, within the density limit", "Cafes", TextRange(102, 107)),
     ))
   }
+
+  List(
+    ("\"", "\""),
+    ("'", "'"),
+    ("`", "`"),
+    ("{", "}"),
+    ("[", "]"),
+    ("(", ")"),
+    ("“", "”"),
+    ("‘", "’)"),
+    ("-", ""),
+    ("–", ""),
+    ("—", "")
+  ).foreach {
+    case (leftNonWordChar, rightNonWordChar) =>
```

Comment:
Out of interest, why do we need the "rightNonWordChar" for this test?

Reply:
Mm, good point – we don't! It just felt a bit silly to provide something which didn't look like proper text – for example, w/ open parens. Arguably this is just confusing for users, happy to get rid – or provide a comment to explain.

Reply:
I would lean towards removing them, but it's not a big deal if you want to leave it.

Reply:
Yeah, now you've mentioned it I agree. I'll do that.